### PR TITLE
Lots of cleanup (related to Muon) which should help users :)

### PR DESCRIPTION
**Pull request by bsclifton**
Fixes https://github.com/brave/browser-laptop/issues/5731

Auditors: @jkup, @bbondy, @bridiver, @jonathansampson

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

### CODE REVIEW

**Comment by bsclifton** [comment]
Please also see updates made here:
https://github.com/brave/browser-laptop/wiki

and here:
https://github.com/brave/browser-laptop-bootstrap/wiki

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ For other versions of our browser, please see:
 
 To download the latest release, [see our releases page](https://github.com/brave/browser-laptop/releases).
 
-For a more user-friendly download page, [please visit our website](https://brave.com/downloads.html).
+You can also [visit our website](https://brave.com/downloads.html) to get the latest stable release (along with a more user-friendly download page).
```
Comment (jonathansampson) [inline README.md]: We also appear to support https://brave.com/downloads, without the .html extension. May be preferred, aesthetically.

**Comment by jonathansampson** [comment]
LGTM!

**Comment by bbondy** [comment]
Thanks!

**Pull request merged**